torch_glow/tests/nodes/reshape_test.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import torch
+import torch_glow
+
+from tests.utils import jitVsGlow
+
+
+def jitScriptVsGlow(f_torch, f_glow, *inputs):
```

Comment:
Cool, no more tracing. Can we just use this for all tests?

Reply:
I believe we can, since we have already migrated to our own customFuseNode